docker/Dockerfile.psga-pipeline

```diff
@@ -44,6 +44,10 @@ RUN nextflow plugin install nf-amazon@1.11.0 \
   && pip install setuptools==63.1.0 \
   && pip install --progress-bar=off --no-cache-dir --upgrade pip==${PIP_VERSION} \
   && pip install --progress-bar=off --no-cache-dir poetry==${POETRY_VERSION} \
+  # the following uninstall & install commands are a fix for urllib3 being installed by something else
+  # in a version that breaks poetry installation, e.g. see https://github.com/python-poetry/poetry/issues/7878
+  && pip uninstall -y urllib3 \
+  && pip install urllib3==1.26.14 \
```

Comment:
```suggestion
```
Probably don't need this because the Poetry install command will add it as per the config. 

https://github.com/Congenica/psga-pipeline/blob/c5753c4a815a0407424e05fc9121f3ae10a9dd73/pyproject.toml#L21

This is potentially an issue with the debian buster image coming with an opinionated installation of Python. An Alpine image would potentially remove that, although it would introduce a tonne of other missing libraries / binaries.

Reply:
The package should be updated to use a venv / virtualenv installation of python, this kind of issue is why. Also allows the control of which version of python is installed, from what I can see with the base image the version of python could change with no warning / control to block it.

Reply:
Looks to be pinned by python here: https://github.com/Congenica/psga-pipeline/blob/dev/docker/sars_cov_2/psga.yml#L10

A venv would still prevent anything opinionated coming in from the base image that have their own python for the OS.

Reply:
We'll make this change on a new ticket @PatrickJordanCongenica. Ticket to follow.

Reply:
Ticket to repair this: https://jira.congenica.net/browse/PSG-3290